job-dsls/jobs/downstream_pr_jobs.groovy

```diff
@@ -27,11 +27,8 @@ def final DEFAULTS = [
                 "**/target/*.log",
                 "**/target/testStatusListener*",
                 "**/target/screenshots/**",
-                "**/target/kie-wb*wildfly*.war",
-                "**/target/kie-wb*eap*.war",
-                "**/target/kie-drools-wb*wildfly*.war",
-                "**/target/kie-drools-wb*eap*.war",
-                "**/target/kie-server-*ee6.war",
```

Comment:
Removing kie-server ee6 war, as @mbiarnes asked me some time ago (this war is no longer produced on master).